index.html

```diff
@@ -2,10 +2,10 @@
 <html lang="en-US">
     <meta charset="utf-8">
     <title>Redirecting&hellip;</title>
-    <link rel="canonical" href="0.57.5.0/index.html">
-    <script>location="0.57.5.0/index.html"</script>
-    <meta http-equiv="refresh" content="0; url=0.57.5.0/index.html">
+    <link rel="canonical" href="0.59.0.0/index.html">
```

Comment:
Should this change be part of this PR?

Reply:
It should not - thanks for spotting - removed in 97a9c84.